pkgs/tools/misc/ollama/default.nix

```diff
@@ -19,23 +19,24 @@
 , linuxPackages
 , darwin
 
-  # one of `[ null "rocm" "cuda" ]`
-, acceleration ? null
-
 , testers
 , ollama
+
+, config
+  # one of `[ null "rocm" "cuda" ]`
+, acceleration ? null
```

Comment:
I'd consider adding the explicit `false` variant which would produce the cpu-only ollama regardless of nixpkgs' instantiation

Reply:
A side note on this name: if we happen to add a package named "acceleration" at some point, this is going to break eval:) It might be prudent to change it early, say `ollamaAcceleration`

Reply:
I would like to keep the nixos option named the same as the package parameter if at all possible. Are there any rules on package naming across all of nixpkgs? Such as no underscores, only dashes, or anything along those lines?